index.html

```diff
@@ -83,15 +82,15 @@ <h1 id="filter">Recent Site</h1>
             <h2><span id="view">Graph</span> View</h2>
             <!-- @todo display list or graph view, update view based on filter and controls -->
             <!-- @todo remove the hard-coded width, height values -->
-            <canvas id="canvas" width="900" height="300"></canvas>
+            <canvas id="canvas" width="100%" height="100%"></canvas>

```

Comment:
@princiya I updated the canvas size to make it fit inside its parent element. I hope that's all right with you!